aclclient/client.go

```diff
@@ -1,13 +1,13 @@
 package aclclient
 
 import (
-	"github.com/juju/aclstore/params"
+	"github.com/juju/aclstore/v2/params"
 	"golang.org/x/net/context"
 	errgo "gopkg.in/errgo.v1"
 	"gopkg.in/httprequest.v1"
 )
 
-//go:generate httprequest-generate-client github.com/juju/aclstore handler client
+//go:generate httprequest-generate-client github.com/juju/aclstore/v2 handler1 client
```

Comment:
why is this now handler1? a typo or something I've misunderstood?

Reply:
because you changed the name of the type :)